app/src/main/res/values/strings.xml

```diff
@@ -208,6 +208,14 @@
   <string name="appointmentreminder_call_later_in">Call later in</string>
   <string name="appointmentreminder_done">Done</string>
 
+  <!-- Remove Overdue Patients -->
+  <string name="removeoverduepatient_select_a_reason">Select a Reason</string>
+  <string name="removepatient_patient_has_already_visited_clinic">Patient has already visited clinic</string>
```

Comment:
Looks like you've accidentally used different prefixes for strings in the same layout.